layouts/community/ergodox/coderkun_neo3/keymap.c

```diff
@@ -0,0 +1,288 @@
+#include QMK_KEYBOARD_H
+#include "led.h"
+#include "keymap_extras/keymap_german.h"
+#include "keymap_extras/keymap_neo2.h"
+
+// Layer names
+#define BASE 0      // default layer
+#define FMU  1      // FMU layer
+#define PMN  2      // poor man’s Neo (Neo 2)
+#define PMB  3      // poor man’s Bone (Neo 3)
+#define NHL  4      // Neo/Bone’s software layer 4 rebuilt in Hardware
```

Comment:
```suggestion
enum layer_names {
    BASE,       // default layer
    FMU,        // FMU layer
    PMN,        // poor man’s Neo (Neo 2)
    PMB,        // poor man’s Bone (Neo 3)
    NHL,        // Neo/Bone’s software layer 4 rebuilt in Hardware
};
```

`layouts/community/ergodox/coderkun_neo2/keymap.c` could use a similar treatment as well.